templates/base/Dockerfile

```diff
@@ -3,10 +3,28 @@ FROM --platform=${TARGETPLATFORM} ubuntu:noble AS ubuntu-noble
 COPY resources/etc/created-date /etc/created-date
 
 RUN apt-get update && \
-	DEBIAN_FRONTEND=noninteractive TZ=Etc/UTC apt-get install --no-install-recommends --yes adduser bash ca-certificates curl jq less libnss3 telnet tini tree unzip && \
+	DEBIAN_FRONTEND=noninteractive TZ=Etc/UTC apt-get install --no-install-recommends --yes adduser bash build-essential ca-certificates curl jq less libnghttp2-dev libnss3 libssl-dev nghttp2 telnet tini tree unzip && \
 	apt-get upgrade --yes && \
 	apt-get clean
 
+# Fix https://ubuntu.com/security/CVE-2024-2398
+#
+# START
+RUN VERSION=8.7.1 && \
```

Comment:
@Wykenz, can you please move the curl installation to be bellow the inclusion of the `liferay` user?